internal/dag/builder.go

```diff
@@ -753,7 +756,7 @@ func (b *Builder) processIngressRoutes(sw *ObjectStatusWriter, ir *ingressroutev
 			sw.Commit()
 		}
 	}
-	sw.SetValid()
+	sw.SetValid("valid IngressRoute")
```

Comment:
please revert this. The name of the object can be inferred from the type stored in the statuswriter

Reply:
I guess I can make it `WithValue(...)`, but the problem is that `IngressRoute` is hard-coded in the `SetValid()` method: https://github.com/heptio/contour/pull/1482/files#diff-942106236f0db9d26ef4d33d498122eeL66